mods/d2k/rules/structures.yaml

```diff
@@ -1155,7 +1155,7 @@ palace:
 		OrderName: ProduceActorPower.Fremen
 	ProduceActorPower@saboteur:
 		Description: Recruit Saboteur
-		LongDesc: Sneaky infantry, armed with explosives\n  Strong vs Buildings\n  Weak vs Everything\n  Special Ability: destroy buildings
+		LongDesc: Sneaky infantry, armed with explosives.\nCan be deployed to become invisible for 15 seconds.\n  Strong vs Buildings\n  Weak vs Everything\n  Special Ability: destroy buildings
```

Comment:
The specific time will depend on the game speed, so please don't mention it here.